patterns/atoms/media/video/video.twig

```diff
@@ -0,0 +1,4 @@
+<video width="{{video.width}}" height="{{video.height}}" controls="{{video.controls}}" class="{{video.class}}">
```

Comment:
I don't want to put a lot extra work here, seems like it's not critical at the moment. We can leave just a simple TODO and come back later.